stdlib/array.gr

```diff
@@ -242,7 +252,10 @@ provide let copy = array => {
  * @param fn: The iterator function to call with each element
  * @param n: The number of times to iterate the given array
  * @param array: The array to iterate
- *
+ * 
+ * @example
+ * Array.cycle(a => print(a), 2, [> 1, 2, 3]) // sep lines: 1 2 3 1 2 3
```

Comment:
A good example that might alternativly work well for this is something like:
```suggestion
* @example
*  let arr = [> 'a', 'b', 'c', 'd', 'e' ]
*  let mut str = ""
 * Array.cycle(a =>str = str ++ a, 2, arr)
 * assert str == "abcdeabcde"
```

This shows well what cycle is doing I think though. This could also be used on forEach

Again wait for feedback from the core team before applying any of these, if you choose to apply them.

Reply:
Thanks Jake.  Again I think the single line with the embedded print(a) clearly shows to the person learning this function what is happening.  I did use sep lines: on I think three or so examples so I understand this might seem different.  But it shows that these are processed like a for loop when you cycle.  I believe this is an elegant way to demonstrate this in one line of code.